Fixed caps; added discovery endpoint (#65)
mulesoft/docs-access-management: modules/ROOT/pages/configure-client-management-openid-task.adoc

[NOTE]
The Refresh token can be selected only if the authorization grant type is also selected.

> In addition to these scopes, if you have configured the Issuer field while setting up the OIDC Dynamic Client registration provider, Anypoint Platform auto-populates the Anypoint Platform UI with all grant types supported by the provider. This includes grant types such as client credentials, password, etc. To verify the grant types your provider supports, check their discovery endpoint using `$ISSUER/.well-known/openid-configuration`.

Author: ```suggestion
In addition to these scopes, if you have configured the Issuer field while setting up the OIDC Dynamic Client registration provider, Anypoint Platform auto-populates the Anypoint Platform UI with all grant types supported by the provider. This includes grant types such as client credentials, password, etc. To verify the grant types your provider supports, check their discovery endpoint using `$ISSUER/.well-known/openid-configuration`, where `$ISSUER` is the issuer configured as the provider. 
```

> In addition to these scopes, if you have configured the Issuer field while setting up the OIDC Dynamic Client registration provider, Anypoint Platform auto-populates the Anypoint Platform UI with all grant types supported by the provider. This includes grant types such as client credentials, password, etc. To verify the grant types your provider supports, check their discovery endpoint using `$ISSUER/.well-known/openid-configuration`, where `$ISSUER` is the issuer configured as the provider. 

Reviewer: ```suggestion
In addition to these scopes, if you configure the Issuer field while setting up the OIDC Dynamic Client registration provider, Anypoint Platform autopopulates its UI with all grant types supported by the provider. This includes grant types such as client credentials, password, and so on. To verify the grant types your provider supports, check their discovery endpoint using `__$ISSUER__/.well-known/openid-configuration`, where `_$ISSUER_` is the issuer configured as the provider. 
```

> In addition to these scopes, if you have configured the Issuer field while setting up the OIDC Dynamic Client registration provider, Anypoint Platform auto-populates the Anypoint Platform UI with all grant types supported by the provider. This includes grant types such as client credentials, password, etc. To verify the grant types your provider supports, check their discovery endpoint using `$ISSUER/.well-known/openid-configuration`, where `$ISSUER` is the issuer configured as the provider. 

Reviewer 2: ```suggestion
In addition to these scopes, if you have configured the Issuer field while setting up the OIDC Dynamic Client registration provider, Anypoint Platform auto-populates the Anypoint Platform UI with all grant types supported by the provider. This includes grant types such as client credentials, password, and so on. To verify the grant types your provider supports, check their discovery endpoint using `$ISSUER/.well-known/openid-configuration`, where `$ISSUER` is the issuer configured as the provider. 
```
